sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -152,7 +169,7 @@ public static void VerifyNoIssueReported(Compilation compilation, DiagnosticAnal
 
             if (expectedIssue == null)
             {
-                Execute.Assertion.FailWith($"Issue with message '{message}' not expected on line {lineNumber}");
+                Execute.Assertion.FailWith($"Issue with message '{(string.IsNullOrEmpty(extraInfo) ? message : extraInfo)}' not expected on line {lineNumber}");
```

Comment:
_extraInfo_ is never empty so you could just use the null coalesce operator: _extraInfo ?? message_